packages/gatsby-theme-carbon/src/components/MDXProvider.js

```diff
@@ -3,6 +3,22 @@ import { MDXProvider as Provider } from '@mdx-js/react';
 import { P, H1, H2, H3, H4, H5, Ul, Ol } from './markdown';
 import PageTable from './PageTable';
 import Code from './Code';
+import {
+  PageDescription,
+  AnchorLinks,
+  AnchorLink,
+  Video,
+  DoDontExample,
+  Row,
+  Column,
+  Grid,
+  Caption,
+  ResourceCard,
+  ArticleCard,
+  Aside,
+  FeatureCard,
+  ImageCard,
+} from 'gatsby-theme-carbon';
```

Comment:
I think this might be the issue. You're attempting to import from `gatsby-theme-carbon` from within `gatasby-theme-carbon`. Maybe try just using relative imports.

Reply:
🤦‍♂ thanks. this is what happens when I'm switching between three different projects in one day. lol